src/source/Threadpool/ThreadpoolContext.h

```diff
@@ -0,0 +1,29 @@
+/*******************************************
+Main internal include file
+*******************************************/
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_THREADPOOLCONTEXT__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_THREADPOOLCONTEXT__
+
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+////////////////////////////////////////////////////
+// Project include files
+////////////////////////////////////////////////////
+
+typedef struct {
+    PThreadpool pThreadpool;
```

Comment:
This needs an initialized flag.

Reply:
Had it initially, but removed it because there is a NULL check on threadpool object in every API. If it is NULL, it is uninitialized, so found to have an initializer flag redundant. Any other reason you foresee we would need the flag?

Reply:
Done